projects/sandbox/train/train/train.py

```diff
@@ -266,10 +245,14 @@ def main(
             tracker,
             valid_background,
             valid_waveforms,
-            sample_rate,
+            psd_estimator=psd_estimator,
+            whitener=whitener,
+            snr_thresh=snr_thresh,
+            highpass=highpass,
+            sample_rate=sample_rate,
             stride=valid_stride,
             injection_stride=4,
-            kernel_length=kernel_length,
+            kernel_length=sample_length,
```

Comment:
Worth thinking about how to get our naming conventions clear and consistent here, though certainly not top priority.